src/components/atoms/ActionButton.js

```diff
@@ -19,13 +23,22 @@ export function ActionButton(props) {
       onMouseLeave={props.onMouseLeave}
       id={props.id}
     >
+      {props.icon}
```

Comment:
better to have the span element here if the icon class is passed in 

`{props.icon ? <span className={props.icon}/>: undefined} `

Reply:
updated